src/components/BaseLayerSwitcher/BaseLayerSwitcher.js

```diff
@@ -109,6 +109,15 @@ function BaseLayerSwitcher({
 
   const nextImage = getNextImage(currentLayer, baseLayers, images);
 
+  useEffect(() => {
```

Comment:
Why is this in a `useEffect` hook? Why do you rename `baselayer`? Why do you use a `map` function without doing something with the result?

Reply:
I want to assign the corresponding image to each layer componentDidMount-style, so only the first time the component renders, which is why it is a useEffect with an empty array of dependencies. forEach instead of map would suffice here, I agree

Reply:
We already map over the baseLayers in line 165 so we can do this stuff directly there. Should be pretty much the same in terms of performance and will be cleaner.

Reply:
Removed this hook